### ISSUE TITLE
Scrum Update 5

### ISSUE THREAD

**Issue by tommytwm** [aleemer/Internify#83]
**p8f2b**
- The past sprint I've contributed on polishing the endpoints and mutations for the Create feature, such as making and handling bulk query calls, update/edit calls, as well as making sure that the Profile page was able to query all the necessary information and handle redirects appropriately.
- Some things that could be done better would include doing better research on frameworks like [jsPDF](https://github.com/MrRio/jsPDF) so that we could be more confident in implementing features like being able to export to PDF.
- For the next two weeks we will be working on our stretch goals to implement the prefills/templates page as well as polishing up final bugs and features that will need to me met before our final presentation

**Comment by ghost** [comment]
h9h1b

- This past spring, I've contributed to the development of the matchesReducer, feedback component, matches backend model, and view page; this includes assisting @ReneReid with the development of the matches algorithm and conditional rendering of student feedback, and also building out the models backend
- One thing I could have done better is putting all the excess functions in an effects file.
- For the next two weeks, I'll finish the backend for matches, add feedback information to the view page, and assist with any other features/final bugs that we will need to complete before our final presentation.

**Comment by JudeShamsi** [comment]
g3j2b

The past sprint I assisted with implementing the profile page and create page APIs. I refactored the code to remove unused and duplicated code, as well as utilize redux to handle state instead of react. I created the registered keys component and created the Prefill component page. 
For the next two weeks I will implement the prefills/templates page and finish handling the state management of checkboxes and radio buttons in the job posting pages using redux. We will also fix any leftover bugs and refactor tech debt.

**Comment by ReneReid** [comment]
s6a3b

**Last Sprint**

1. Helped with redesigning the notes component to display more relevant data (dis-aggregated match information)
2. Helped with implementing notes section, make it progressively update with each step of Create
3. Helped update the matches reducer (match a job posting to the students)
4. Helped build out the job edit page (not complete - faced merge conflicts with develop)

**Next Sprint**

1. Finish job edit page (resolve merge conflicts correctly)
2. Assist with template pages
3. Assist with any bugs
